src/notebook/output-area/widget.ts

```diff
@@ -442,6 +442,9 @@ namespace OutputAreaWidget {
     createOutput(): Widget {
       let widget = new Widget();
       widget.layout = new PanelLayout();
+      let prompt = new Widget();
+      prompt.addClass(PROMPT_CLASS);
+      (widget.layout as PanelLayout).addChild(prompt);
```

Comment:
create layout, add child to layout, then set layout on parent widget - this avoids the cast